README.md

```diff
@@ -1,7 +1,7 @@
 # Appcast library
 
 > Sparkle uses appcasts to get information about software updates.
-> An appcast is an RSS feed with some extra information for Sparkle’s purposes.
+> An appcast is wan RSS feed with some extra information for Sparkle’s purposes.
```

Comment:
```suggestion
> An appcast is an RSS feed with some extra information for Sparkle’s purposes.
```

Reply:
Thanks for notifying me, this change should not have been commited.